l10n_it_fatturapa_out_wt/wizard/wizard_export_fatturapa.py

```diff
@@ -1,127 +1,79 @@
-from odoo import models
-from odoo.exceptions import Warning as UserError
+from odoo import api, models
 from odoo.tools.float_utils import float_round
-from odoo.tools.translate import _
 
-from odoo.addons.l10n_it_fatturapa.bindings.fatturapa import (
-    DatiCassaPrevidenzialeType,
-    DatiRiepilogoType,
-    DatiRitenutaType,
+from odoo.addons.l10n_it_account.tools.account_tools import encode_for_export
+from odoo.addons.l10n_it_fatturapa_out.wizard.wizard_export_fatturapa import (
+    format_numbers,
 )
 
 WT_TAX_CODE = {"inps": "RT03", "enasarco": "RT04", "enpam": "RT05", "other": "RT06"}
 
-TC_CODE = {
-    "inps": "TC22",
-    "enasarco": "TC07",
-    "enpam": "TC09",
-}
 
+def get_withholding_type(wt_types, partner):
```

Comment:
Lo trasformerei in un metodo (con `api.model`) della classe `WizardExportFatturapa`, analogamente a quanto fatto [qui](https://github.com/OCA/l10n-italy/blob/e4d287bee8ccbe3916f12fcbc5fade0377105e5e/l10n_it_fatturapa_out_rc/wizard/wizard_export_fatturapa.py#L12).

Reply:
Fatto.